jans-cedarling/bindings/cedarling_uniffi/androidApp/app/src/main/java/com/example/androidapp/views/JsonTreeView.kt

```
@@ -15,9 +15,20 @@ import com.google.gson.JsonParser
 
 @Composable
 fun JsonTreeView(jsonString: String) {
-    val jsonElement = remember { JsonParser.parseString(jsonString) }
+    val jsonElement = remember(jsonString) {
+        try {
+            JsonParser.parseString(jsonString)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)